core/src/test/java/io/questdb/griffin/wal/WalWriterFuzzTest.java

```diff
@@ -454,7 +446,7 @@ private ObjList<FuzzTransaction> createTransactions(Rnd rnd, String tableNameBas
                     start,
                     end,
                     fuzzRowCount,
-                    isO3,
+                    transactionCount, isO3,
```

Comment:
new line missing

Reply:
fixed in https://github.com/questdb/questdb/pull/2759/commits/be85cbfb3d59b70be8180622869ea92a58033841